components/eam/src/chemistry/pp_linoz_mam4_resus_mom_soag/mo_rxt_rates_conv.F90

```diff
@@ -4,16 +4,17 @@ module mo_rxt_rates_conv
   private
   public :: set_rates
 contains
-   subroutine set_rates( rxt_rates, sol, ncol )
-      real(r8), intent(inout) :: rxt_rates(:,:,:)
-      real(r8), intent(in) :: sol(:,:,:)
-      integer, intent(in) :: ncol
-      rxt_rates(:ncol,:,     1) = rxt_rates(:ncol,:,     1)*sol(:ncol,:,     2)                                                ! rate_const*H2O2
```

Comment:
maybe remove the excessive whitespaces as well?

Reply:
I have removed these white spaces.